cmd/manager/main.go

```diff
@@ -1,39 +1,43 @@
 package main
 
 import (
+	"context"
 	"flag"
 	"fmt"
 	"os"
 	"runtime"
 
+	"github.com/gluster/anthill/pkg/apis"
+	"github.com/gluster/anthill/pkg/controller"
 	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
```

Comment:
can we add a new line here to seperate out the import orders.

(
"go pkgs"

"antill pkgs"

"third-party pkgs"
)

Reply:
This gets overwritten each time we upgrade the sdk. I'm fine w/ changing it in files we manage, but repeatedly doing it here just makes upgrades more painful.